prow/integ-suite-ocp.sh

```diff
@@ -124,5 +126,10 @@ if [ "${TEST_OUTPUT_FORMAT}" == "junit" ]; then
     base_cmd+=" 2>&1 | tee >(${JUNIT_REPORT} > ${ARTIFACTS_DIR}/junit/junit.xml)"
 fi
 
+# unset DOCKER_CONFIG var to avoid credentials error when helm test is running
+if [ "${TEST_SUITE}" == "helm" ]; then
+    unset DOCKER_CONFIG
```

Comment:
Perhaps safer to `export DOCKER_CONFIG=/dev/null` or similar, since we don't know for sure whether the build in prow is setting this or it's using the default location